src/PerfView/Dialogs/RunCommandDialog.xaml

```diff
@@ -6,507 +6,494 @@
         Style="{DynamicResource CustomWindowStyle}"
         Background="{StaticResource ControlDarkerBackground}"
         Title="Collecting ETW Data while running a command"
-        Height="304" Width="900" MinHeight="210" MinWidth="900">
+        Height="304" Width="950" MinHeight="210" MinWidth="950">
```

Comment:
Is this change od width(s) intentional?

Reply:
Yes, this is being done because I've now made the run command dialog scrollable and the scrollbar needs an extra 50 pixels. Below I compare new (on top), and previous (on bottom)

![image](https://github.com/user-attachments/assets/eb1e0566-d401-4821-8547-0fd2e2f88f8a)